### PR TITLE
chore: use gunicorn on prod containers

### PR DESCRIPTION
**Pull request by ahoneiser**

### CODE REVIEW

File: images/app/Dockerfile [hunk 1]
```diff
@@ -28,4 +28,4 @@ USER app
 
 ENV PATH=/.venv/bin:${PATH}
 
-CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8000"]
+CMD ["gunicorn", "--bind", "0.0.0.0:8000", "--worker-class", "uvicorn.workers.UvicornWorker", "--workers", "4", "app.main:app"]
```
Comment (bkochendorfer) [inline images/app/Dockerfile:31]: :thinking: It would be cool if workers was also configurable from an environment variable. I wonder if that is possible

Comment (ahoneiser) [reply images/app/Dockerfile:31]: I'm currently adjusting on the deployment, if necessary, but could find a way here of course :ok_hand:

Comment (ahoneiser) [reply images/app/Dockerfile]: okay @bkochendorfer, starting gunicorn with `config.py` now, building settings with pydantic :slightly_smiling_face:

**Review by bkochendorfer — APPROVED** [review]
Assuming that int 4 can be over ridden easily that seems good to me. We generally use an environment variable to tweak these easily on the mozmeao side.

**Review by cbguder — APPROVED** [review]

**Review by bkochendorfer — APPROVED** [review]

**Pull request merged**